utils/actions/detectConsoleLogs.ts

```diff
@@ -109,63 +84,49 @@ export default async function detectConsoleLogs({
   const commentPromises = diffFiles.map(async (file) => {
     const { additions } = getLineDiffs(file.patch ?? "");
 
-    // detect if the additions contain console logs or not
-    try {
-      return await openai
-        .createChatCompletion({
-          model: "gpt-4-1106-preview",
-          messages: [
-            {
-              role: "system",
-              content: `${consoleLogDetectionPrompt} \n ${additions}`,
-            },
-          ],
-        })
-        .then((result) => {
-          const openAIResult =
-            result.data.choices[0].message.content.split(",");
-
-          const addtionsHaveConsoleLog = openAIResult[0];
-          const individualLine = openAIResult[1];
-
-          if (addtionsHaveConsoleLog === "true") {
-            const commentFileDiff = () => {
-              const consoleLogPosition = getConsoleLogPosition({
-                filePatch: file.patch ?? "",
-                individualLine,
-              });
-
-              return octokit
-                .request(
-                  "POST /repos/{owner}/{repo}/pulls/{pull_number}/reviews",
-                  {
-                    owner,
-                    repo,
-                    pull_number: issue_number,
-                    commit_id:
-                      typeof latestCommitHash === "string"
-                        ? latestCommitHash
-                        : undefined,
-                    event: "COMMENT",
-                    path: file.filename,
-                    comments: [
-                      {
-                        path: file.filename,
-                        position: consoleLogPosition || 1, // comment at the beggining of the file by default
-                        body: commentBody,
-                      },
-                    ],
-                  }
-                )
-                .catch((err) => {
-                  console.log(err);
-                });
-            };
-
-            commentFileDiff();
-          }
-        });
-    } catch {}
+    const splitAdditions = additions.split("\n");
+
+    // RegEx to detect console log equivalents in different languages
+    const consoleLogRegex = /(console\.log|print|printf|fmt\.Print|log\.Print|NSLog|puts|println|println!)\([^)]*\)(?![^]*?\/\/|[^]*?\/\*|#)/;
```

Comment:
Could we make an external list?
Adding to a single line seems harder

Reply:
What do you mean by an external list, exactly?

Reply:
```suggestion
const consoleLogTexts = ["console.log", "printLn"]
```

Reply:
And then we use a template string to add it to the Regex

Reply:
It's a valid suggestion. If I'm free of tasks during the holidays I'll improve the readability. Soft promise.